components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/ClientUtils.cs

```diff
@@ -0,0 +1,671 @@
+﻿// ------------------------------------------------------------
```

Comment:
Hi @peterbryntesson, for what are these utils needed?

Reply:
I need to get all fields in the specified type and base types, so I can autopopulate a select clause.

Reply:
I'm sure there is a better way to do this, I'm no expert on the OPC UA SDK. I guess you are the expert :-)

Reply:
@mregen updated the code to sue the NodeCache. Take a look at my latest commit.